src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,404 @@
+    for source, target, relation and pubmed_id.
+
+    :param df: intact dataframe to be preprocessed
+    :return: dataframe with IntAct information
+    """
+    print('Dataframe is being created from the file.')
+
+    # take relevant columns for source, target, relation and PubMed ID
+    df = df[[SOURCE, TARGET, RELATION, PUBMED_ID]]
+
+    # drop nan value rows for interactor B
+    df = df[df[TARGET] != '-', :]
+
+    return df
+
+
+def split_to_list(unsplit_list: str, separator: str = '|') -> List:
+    """Split a list of strings that contains multiple values that are separated by a defined separator into a list of lists.
+
+    :param unsplit_list: list of strings to be splitted
+    :param separator: separator between elements
+    :return: list of lists of splitted elements
+    """
+    return [x.split(separator) for x in unsplit_list]
+
+
+def split_column_str_to_list(df: pd.DataFrame, column_name: str) -> pd.DataFrame:
+    """Split the values of a column that has a string containing multiple values by a separator.
+
+    :param df: dataframe with string to be splitted
+    :param column_name: column name of string to be splitted
+    :return: dataframe with list of splitted elements
+    """
+    list_column = df.loc[:, column_name]
+    splitted_lists = split_to_list(list_column, separator='|')
+
+    df.loc[:, column_name] = splitted_lists
+
+    return df
+
+
+def list_pubmed(publication_ids: Iterable[str]) -> List[str]:
+    """Filter the publication ids for Pubmed IDs.
+
+    :param publication_ids: list of publication ids
+    :return: filtered list of pubmed ids
+    """
+    final_list = []
+    for publications in publication_ids:
+        publications_list = split_to_list(publications, separator='|')
+        flag = False
+        for i in publications_list:
+            if i.startswith('pubmed:'):
+                final_list.append(i)
+                flag = True
+        if not flag:
+            final_list.append('no pubmed id')
+
+    return final_list
+
+
+def filter_for_pubmed(df: pd.DataFrame, column_name: str):
+    """Filter the publication ids for pubmed ids.
+
+    :param df: dataframe
+    :param column_name: column with publication ids
+    :return: dataframe with filtered column
+    """
+    pubmed_list = list_pubmed(df.loc[:, column_name])
+    df = add_to_df(df=df, column_name=column_name, list_to_add=pubmed_list)
+    return df
+
+
+def add_to_df(df: pd.DataFrame, column_name: str, list_to_add: List) -> pd.DataFrame:
+    """Add a column to a dataframe.
+
+    :param df: dataframe
+    :param column_name: column name
+    :param list_to_add: list to be added in column in the dataframe
+    :return: df with updated column
+    """
+    df.loc[:, column_name] = list_to_add
+    return df
+
+
+def get_processed_intact_df() -> pd.DataFrame:
+    """Load, filter and rename intact dataframe.
+
+    :return: processed dataframe
+    """
+    print(_load_file())
```

Comment:
no prints allowed. use python logging if you must

Reply:
Was just to test, because loading the file into pandas still does not work